test/helpers/browser.js

```diff
@@ -7,6 +7,7 @@ var vm = require('vm');
 
 function createContext() {
   var context = {
+    self: this,
```

Comment:
@davidcheung please fix the code too, `self` should be set to `context` here. You have also typos in the commit message, please re-read the message and fix them.